spec/requests/employee_invitation_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe 'EmployeeInvitationController', type: :request do
+  let(:user) { create(:user, :admin) }
+  let(:not_listed_user) { create(:user) }
+  let(:company) { create(:company) }
+  let(:headers) { { Authorization: sign_in(user) } }
+
+  describe 'POST /companies/:company_id/employee_invitations' do
+    context 'when user is no registered on Comarev' do
+      subject(:invitation_controller) do
+        post company_employee_invitation_path(company), params: { email: 'test_email@test.com' },
+          headers: headers
+      end
+
+      it 'Returns status ok' do
+        invitation_controller
+
+        expect(response).to have_http_status(:ok)
+      end
+    end
+
+    context 'when user already is listed on a Company' do
+      subject(:invitation_controller) do
+        post company_employee_invitation_path(company), params: { email: user.email },
+          headers: headers
+      end
+
+      before do
+        create(:company_user, user: user, company: company)
+      end
+
+      it 'Returns status unprocessable entity' do
+        invitation_controller
+
+        expect(response).to have_http_status(:unprocessable_entity)
+      end
+    end
+
+    context 'when user is not listed on a Company' do
+      subject(:invitation_controller) do
+        post company_employee_invitation_path(company), params: { email: not_listed_user.email },
+          headers: headers
```

Comment:
I think it looks better this way when you have args and kwargs
```suggestion
        post company_employee_invitation_path(company), params: { email: not_listed_user.email },
                                                        headers: headers
```

Reply:
How can I replace the `params` block using `args` ?